### ISSUE TITLE
[Subscriptions] Admin setting to change default value of user "Automatically follow discussions that I reply" setting 

### ISSUE THREAD

**Issue by tobyzerner** [flarum/core#894]
As per https://discuss.flarum.org/d/372-should-you-be-following-your-own-post/10

**Comment by haydenjames** [comment]
What very of beta will this show up in?

**Comment by rsolvang** [comment]
Any updates on this? It would be great to set the default behavior for all users in the admin panel!

**Comment by gurjyot** [comment]
I would say to have two different notification types in this.
1. Follow the discussion I started.
2. Follow the discussion I replied.

Since, in the first one, the user actually wants to get notified of the replies and that's the whole point of starting a discussion. Or he wants to be notified of all the replies after his replies then, he can use the "follow" feature as that feature is there for the same reason.
In the second one, the user will start to get a lot of notifications to all the discussions he ever replied or created.
Basically, in the first case, I am just getting the notifications for the discussions I care about. In the second case, I am bombarded with the notification from every discussion I ever participated. And users will always have the chance to enable both the settings.
